src/main/java/org/quantumbadger/redreader/activities/MainActivity.java

```diff
@@ -573,13 +651,26 @@ protected void doRefresh(final RefreshableFragment which, final boolean force, f
 			final View layout;
 
 			if(twoPane) {
-				layout = getLayoutInflater().inflate(R.layout.main_double, null);
-				mLeftPane = (FrameLayout)layout.findViewById(R.id.main_left_frame);
+				if(PrefsUtility.appearance_navigation_type(this, sharedPreferences).equals("single_list")) {
```

Comment:
The string literals "single_list" and "drawer_tabs" are repeated throughout the codebase -- `PrefsUtility` should return an enum value to avoid this.